app/scripts/controllers/edit/yaml.js

```diff
@@ -108,7 +108,7 @@ angular.module('openshiftConsole')
               kind: resource.kind
             }, resource);
```

Comment:
You can remove this block if you're sorting keys using jsyaml below. (Sorry, should have caught on previous review.)